wpcsite/settings.py

```diff
@@ -69,6 +70,14 @@ def rel_path(*subpaths):
     return path.join(BASE_DIR, *subpaths)
 
 
+def env_to_bool(environ_key, default):
```

Comment:
Added [bug 1218237](https://bugzilla.mozilla.org/show_bug.cgi?id=1218237) to track python-decouple feature. I was going to go ahead and write ``env_to_list``, etc. but using one of these packages seems like a better path to standardization, especially if we share the same solution in kuma.  I would like to research this and other options for [configuration](https://www.djangopackages.com/grids/g/configuration/), to make sure we pick a good option.